e2e-environment/Dockerfile

```diff
@@ -24,7 +24,7 @@ FROM python:3.7 as synapse-builder
 ARG SYNAPSE_VERSION
 
 RUN python -m venv /synapse-venv \
-    && /synapse-venv/bin/pip install "matrix-synapse[sqlite]==${SYNAPSE_VERSION}" \
+    && /synapse-venv/bin/pip install "matrix-synapse==${SYNAPSE_VERSION}" \
```

Comment:
Which database is Synapse using per default? What was again the issue with SQLite?

Reply:
Seems like the default is sqlite. Here `pip`was complaining that `sqlite` was not a valid option name.